readthedocs/projects/version_handling.py

```diff
@@ -109,7 +113,8 @@ def version_windows(versions, major=1, minor=1, point=1):
 def parse_version_failsafe(version_string):
     try:
         return Version(
-            unicodedata.normalize('NFKD', unicode(version_string)).encode('ascii', 'ignore')
+            six.text_type(
+                unicodedata.normalize('NFKD', six.text_type(version_string)))
```

Comment:
I believe I cribbed this from Eric's contributions to the original PR. Perhaps he can weigh in. @ericholscher ?

My original understanding of the logic was that we were attempting to drop non-ASCII characters from the version string, then turning it into a normalized Unicode string before passing it to `Version` for parsing. I agree that I don't believe the new logic does that, but I was never certain my original reading was correct.